array-map/main.js

```diff
@@ -0,0 +1,38 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const multiplied = number => {
+  return number * 2;
+};
+
+const doubled = numbers.map(multiplied);
+console.log('Multiplied by 2', doubled);
+
+const price = number => {
+  const formatted = '$' + number.toFixed(2).toString();
```

Comment:
It isn't necessary to use `toString` here, `'$' + number.toFixed(2)` would produce the same result.